examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1181 @@
+{
```

Comment:
<p>Do you really want to feature  remote encryption? Maybe just mention it as a side note as it makes just more code.</p><p>Also, if you want to use encryption_kwargs, use it on all cells also the previous one for example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Concerning showcasing the remote encryption an alternative would also be that you include the local encryption part in your tutorial Part 11 bis - given of course you also think that this direct comparison would make sense and doesn't yet exist in a notebook. :+1</p>